docs/reference/topology-updater-configuration-reference.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Topology-Updater config reference"
+layout: default
+sort: 3
+---
+
+# Configuration file reference of nfd-topology-updater
+{: .no_toc}
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+---
+
+See the
+[sample configuration file](https://github.com/kubernetes-sigs/node-feature-discovery/blob/{{site.release}}/deployment/components/topology-updater-config/nfd-topology-updater.conf.example)
+for a full example configuration.
+
+## excludeList
+
+The `excludeList` specifies a key-value map of allocated resources
+that should not be examined by the topology-updater
+agent per node.
+Each key is a node name with a value as a list of resources
+that should not be examined by the agent for that specific node.
```

Comment:
Really good that you added the config file reference 👍 One suggestion here is to add a **Default:** and **Example:** fields similar to what we have in `docs/reference/worker-commandline-reference.md`. I think with an example use can quickly grasp the format and do her own. WDYT?